pontoon/base/admin.py

```diff
@@ -230,14 +229,14 @@ def enabled(self, obj):
         ExternalProjectResourceInline,
     )
 
-    def save(self, obj):
-        if "visibility" in self.changed_data or "disabled" in self.changed_data:
-            for locale in obj.project_locale.all():
-                locale.aggregate_stats()
+    def save_model(self, request, obj, form, change):
```

Comment:
Hmmmmmm... Of course, now that you aren't using Project.save(), you need to repeat the same code twice. :-(

I think that's even worse than using "disabled_changed" in Project.save(), because it will lead to inconsistencies. You've already introduced one by flipping the order of conditions in the if clause. :-)

So I'd restore Project.save() and move all the logic there. That way we'll have everything in one place and the changeset will be smaller.